### PR TITLE
[Lock] Add --add-platform

### PR DESCRIPTION
**Pull request by segiddins**
\c @indirect @RochesterinNYC

### CODE REVIEW

**Comment by indirect** [comment]
@homu r+

File: lib/bundler/definition.rb [hunk 1]
```diff
@@ -433,6 +432,11 @@ def validate_ruby!
       end
     end
 
+    def add_platform(platform)
+      @new_platform ||= @platforms.include?(platform)
```
Comment (segiddins) [inline lib/bundler/definition.rb]: !

Comment (indirect) [inline lib/bundler/definition.rb]: yeeeah that doesn't look right 😬

**Comment by segiddins** [comment]
Fixed!

**Comment by indirect** [comment]
@homu r+

**Pull request merged**